packages/ui-components/jsconfig.json

```
@@ -0,0 +1,19 @@
+// This file can contain .js-specific Typescript compiler config.
+{
+  "compilerOptions": {
+    "target": "es2020",
```

Just a tweak (I think you might have copied `jsconfig.json` before #2550).

```suggestion
    "target": "esnext",
```